assets/styl/_base/_form-elements.styl

```diff
@@ -121,8 +118,61 @@ input[type='image']
     border 0
     width auto
 
-// iframe should be block by default
-iframe
-    display block
-    width 100%
 
+// Radio buttons
+//
+// Default styles for radio buttons.
+// Styling method takes a lot of inspiration from Materialize CSS.
+//
+// Markup: form-radiobutton.twig
+//
+// Styleguide: Components.Forms.RadioButtons
+
+// Checkboxes
+//
+// Default styles for checkboxes.
+// Styling method takes a lot of inspiration from Materialize CSS.
+//
+// Markup: form-checkbox.twig
+//
+// Styleguide: Components.Forms.Checkboxes
+input[type='checkbox']
+input[type='radio']
+    position absolute
```

Comment:
Position is ok in parent container as no `top|right|bottom|left` is defined.
I added `focus` style on the label to make navigation with keyboard accessible.